tests/ImportOrderCaseSensitive/__snapshots__/ppsi.spec.ts.snap

```diff
@@ -0,0 +1,20 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`mixed-case.ts - typescript-verify > mixed-case.ts 1`] = `
+
+import c from "c";
+import b from "b";
+import a from "a";
+
+import {cat, Dog} from "C";
+import B from "B";
+import A from "A";
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+import a from "a";
+import A from "A";
+import b from "b";
+import B from "B";
+import c from "c";
+import { cat, Dog } from "C";
```

Comment:
I added an end-to-end test, and it looks like this is not behaving as expected.  Would you mind looking into it to see what's not quite wired up yet?